### PR TITLE
Add the description about Note event for debugging

### PR DESCRIPTION
**Pull request by enori**
Pull Request for [#3628](https://github.com/open-policy-agent/opa/issues/3628).

### CODE REVIEW

File: docs/content/policy-reference.md [hunk 1]
```diff
@@ -1025,6 +1025,20 @@ net.cidr_contains_matches({["1.1.0.0/16", "foo"], "1.1.2.0/24"}, {"x": "1.1.1.12
 | ------- |-------------|---------------|
 | <span class="opa-keep-it-together">``trace(string)``</span> | ``trace`` outputs the debug message ``string`` as a ``Note`` event in the query explanation. For example, ``trace("Hello There!")`` includes ``Note "Hello There!"`` in the query explanation. To print variables, use sprintf. For example, ``person := "Bob"; trace(sprintf("Hello There! %v", [person]))`` will emit ``Note "Hello There! Bob"``. | ``SDK-dependent`` |
 
+By default, explanations are disabled. The following table summarizes how you can enable tracing:
```
Comment (srenatus) [inline docs/content/policy-reference.md:1028]: This sentence is repeated in line 1034, it seems.

**Review by srenatus — COMMENTED** [review]
Thanks a lot 👏 one copy-edit request ⬇️

**Review by srenatus — DISMISSED** [review]
Thank you! 👏 

With the commits squashed and signed-off, we can merge this right away 🚀

**Comment by srenatus** [comment]
Heya! Looks like the rebase got a bit out of hand. LMK if you need help with the steps involved:

1. git fetch origin # assuming origin is this repo
2. git checkout main # assuming this is your PR's branch, it seems to be
3. git rebase -i origin/main # opens an editor
   1. remove all commits except 2d5f6c3 and cb2dd58
   2. change "pick cb2dd58" into "f cb2dd58",
   3. save and quit editor
4. git status; git log -1 # see if everything is OK
5. git push -f mine # the origin of your fork

**Comment by srenatus** [comment]
Thanks for bearing with me. I'll get this merged quickly 🚀

**Comment by enori** [comment]
I apologize for any inconvenience this may cause you. 
Thanks for merging!

**Review by srenatus — APPROVED** [review]
Thanks for contributing! ❤️

**Pull request merged**